microshift_networking/microshift-cni.adoc

```diff
@@ -0,0 +1,114 @@
+:_content-type: ASSEMBLY
```

Comment:
This was only relocated from different location?

Reply:
Yes, spreading this section out a bit.